homeassistant/components/zoneminder/camera.py

```diff
@@ -83,3 +107,37 @@ def is_recording(self):
     def available(self):
         """Return True if entity is available."""
         return self._is_available
+
+    @property
+    def is_on(self):
+        """Return true if on."""
+        return self._function != MonitorState.NONE
+
+    @property
+    def motion_detection_enabled(self):
+        """Return the camera motion detection status."""
+        return self._function in (MonitorState.MODECT, MonitorState.MOCORD)
+
+    @property
+    def state_attributes(self):
```

Comment:
We prefer to have separate sensor entities instead of state attributes. Alternatively if the attributes are events we can fire custom events not from a platform. But we should be mindful of breaking more than needed.